frontend/components/pactaversion/Selector.vue

```diff
@@ -0,0 +1,37 @@
+<script setup lang="ts">
+interface Props {
+  value: string | undefined
+}
+const props = defineProps<Props>()
+const emits = defineEmits<(e: 'update:value', value: string | undefined) => void>()
+const value = computed({
+  get: () => props.value,
+  set: (v: string | undefined) => { emits('update:value', v) },
+})
+
+const pactaClient = await usePACTA()
+const prefix = 'components/pactaversion/Selector'
+const { data: pactaVersions, refresh } = await useSimpleAsyncData(
```

Comment:
Loading data in components is one of those _kinda_ anti-patterns we've done in the past that I'd like us to move away from. Passing in `pactaVersions` as a prop and loading it in the relevant page means we don't have a dozen components loading semi-overlapping data on page load.

This is especially important since Nuxt/your useSimpleAsyncData doesn't do caching. So, for example, N `PactaversionSelector`s on a page would issue N `pactaClient.listPactaVersions` calls